ssl/ssl_lib.c

```diff
@@ -3956,9 +4016,47 @@ SSL_CTX *SSL_CTX_new_ex(OSSL_LIB_CTX *libctx, const char *propq,
 
     ssl_ctx_system_config(ret);
 
+#ifndef OPENSSL_NO_SSLKEYLOG
+    if (keylogfile == NULL || *keylogfile == '\0')
+        return ret;
+
+    /* Make sure we have a global lock allocated */
+    if (!RUN_ONCE(&ssl_keylog_once, ssl_keylog_init)) {
+        /* use a trace message as a warning */
+        OSSL_TRACE(TLS, "Unable to initalize keylog data\n");
```

Comment:
I forgot: this also needs rethinking once OSSL_TRACE goes away.

Reply:
Yeah, all those TRACE messages need to be handled.

Reply:
Let's deal with this in the no-trace PR.